docs/release.md

```diff
@@ -0,0 +1,81 @@
+# Release Process
+
+This document outlines the release process of the load-test tool. It is codified in [a Playbook](https://community.mattermost.com/playbooks/playbooks/95pa6a8fcp8p8xsn4frqejcide) that the team follows monthly in the Community server. If you don't have access to the playbook, please ping the team.
+
+## Requirements
+
+This release process follows [the release process of the self-hosted version of Mattermost](https://handbook.mattermost.com/operations/research-and-development/product/release-process/feature-release).
+
+The requirements for this release process are the following:
+- The monthly performance comparisons, where we compare the Mattermost version about to be released with the latest stable one, are based on the RC-1 version of the new release.
```

Comment:
Just some random thought at this point but have we ever considered doing an extra comparison for ESRs to be compared with the previous ESRs?

Reply:
That's something that's been on my mind for a while, but we've never planned anything related. We probably should at some point.

Reply:
I agree with the idea of adding loadtesting to compare ESRs.